pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.Ericsson</groupId>
+    <artifactId>eiffel-remrem-shared</artifactId>
+    <version>0.3.7</version>
+    <properties>
+        <junit.version>4.12</junit.version>
+        <google-gson.version>2.8.1</google-gson.version>
+        <commons-codec.version>1.10</commons-codec.version>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+        <dependency>
```

Comment:
This one also we can remove, 
Only one class we are using JsonParser from this library, in VersionService.java. 
and it is a just Object declaration, so please remove

Reply:
okie